src/metrics.rs

```diff
@@ -32,3 +33,64 @@ pub use print::Print;
 async fn all_repos(org: &octocrab::orgs::OrgHandler<'_>) -> Vec<octocrab::models::Repository> {
     util::accumulate_pages(|page| org.list_repos().page(page).send()).await?
 }
+
+#[derive(Deserialize, Debug)]
+pub struct Response<T> {
+    data: T,
+    errors: Option<Vec<serde_json::Value>>,
+}
+
+#[derive(Deserialize, Debug)]
+struct AllReposData {
+    organization: Organization,
+}
+
+#[derive(Deserialize, Debug)]
+struct Organization {
+    repositories: Repositories,
+}
+
+#[derive(Deserialize, Debug)]
+struct Repositories {
+    edges: Vec<Node>,
+}
+
+#[derive(Deserialize, Debug)]
+struct Node {
+    node: RepoNode,
+}
+
+#[derive(Deserialize, Debug)]
+struct RepoNode {
+    name: String,
+}
+
+#[throws]
+async fn all_repos_graphql(org: &str) -> Vec<String> {
+    let octo = octocrab::instance();
+    let query_string = format!(
+        r#"query {{
+            organization(login:"{org_name}"){{
+                repositories(first:100){{
```

Comment:
does this assume no more than 100 repos?

Reply:
Huge bug, glad you caught it. 🐛